enterprise/server/backends/memcache/memcache.go

```diff
@@ -34,6 +41,23 @@ type Cache struct {
 	prefix string
 }
 
+func Register(env *real_environment.RealEnv) error {
+	if fmt.Sprintf("%T", env.GetCache()) == fmt.Sprintf("%T", composable_cache.NewComposableCache(nil, nil, 0)) {
+		// Cache has already been composed, don't do it again.
+		return nil
+	}
+	if len(*memcacheTargets) == 0 {
+		return nil
+	}
```

Comment:
nit: move up first

Reply:
Done! and did the analogous thing in redis_cache.